cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -15,21 +15,30 @@
  */
 package io.stargate.cql;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import io.stargate.auth.AuthenticationService;
 import io.stargate.core.activator.BaseActivator;
 import io.stargate.core.metrics.api.Metrics;
 import io.stargate.cql.impl.CqlImpl;
 import io.stargate.db.DbActivator;
 import io.stargate.db.Persistence;
 import io.stargate.db.metrics.api.ClientInfoMetricsTagProvider;
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
 import java.net.InetAddress;
-import java.net.UnknownHostException;
 import java.util.Arrays;
 import java.util.List;
 import javax.annotation.Nullable;
-import org.apache.cassandra.config.Config;
+import org.apache.cassandra.stargate.config.Config;
 
 public class CqlActivator extends BaseActivator {
+  public static ObjectMapper mapper =
+      new ObjectMapper(new YAMLFactory())
```

Comment:
not a big deal but there's also `YAMLMapper` that constructs `YAMLFactory` (no need to change if no other changes made), which is a sub-class of `ObjectMapper`